packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/source/UrlSource.java

```diff
@@ -7,17 +7,22 @@
 /**
  * information about an url request
  */
-public class UrlData {
+public class UrlSource {
 
     @Id
     private String url;
+
     private Date lastFetched;
+
     private String content;
 
-    public UrlData(String url, Date lastFetched, String content) {
+    private Integer statusCode;
```

Comment:
let's use primitive data types, as long as we can avoid having null values. Is it possible here to use `int`?